fab/sampling_methods/ais_test.py

```diff
@@ -12,6 +12,77 @@
 from fab.utils.plotting import plot_history
 from fab.sampling_methods.base import resample
 
+
+def analytic_alpha_2_div(mean_q: torch.Tensor, mean_p: torch.Tensor) -> torch.Tensor:
+    """Calculate \int p(x)^2/q dx where p and q are unit-variance Gaussians."""
+    return torch.exp(torch.sum(mean_p**2 + mean_q**2 - 2*mean_p*mean_q))
+
+
+def test_ais_num_dist(dim: int = 1,
```

Comment:
A nice test that visualises the quality of AIS with increasing number of intermediate distributions.